benchto-driver/src/main/java/io/trino/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -138,6 +138,13 @@ public Future<?> benchmarkFinished(BenchmarkExecutionResult benchmarkExecutionRe
                     if (benchmarkExecutionResult.getUtcEnd() != null) {
                         builder.withEndTime(benchmarkExecutionResult.getUtcEnd().toInstant());
                     }
+                    // Throughput tests have a different query in every execution, but only one, aggregated execution is saved
+                    // so don't save statements for them.
+                    if (!benchmarkExecutionResult.getBenchmark().isThroughputTest()) {
+                        benchmarkExecutionResult.getExecutions().stream()
+                                .findFirst()
+                                .ifPresent(e -> builder.addAttribute("statement", e.getQueryExecution().getStatement()));
```

Comment:
How this will be used? How much data is it?

Reply:
This is the interpolated query, so this: https://github.com/trinodb/trino/blob/master/testing/trino-benchmark-queries/src/main/resources/sql/presto/tpch/q01.sql with evaluated variables.

I'd like to include this in the final report, as an attachment, for two reasons:
* to avoid having to figure out the correct commit to generate a link to the Trino repo (or a different one)
* to make it easier to show the actual query that was executed, without variables

This is similar to saving the query plan in the query info table.

Reply:
If we're concerned with the size of the attributes table, we could normalize these queries.